packages/puppeteer-core/src/bidi/Browser.ts

```diff
@@ -323,7 +323,12 @@ export class BidiBrowser extends Browser {
     return this.#browserTarget;
   }
 
-  override disconnect(): void {
-    this;
+  override async disconnect(): Promise<void> {
+    try {
+      await this.#connection.send('session.end', {});
+      this.#connection.dispose();
+    } catch (e) {
+      console.log(e);
```

Comment:
Better use `debugError` instead of console.log. Also, we probably should call this.#connection.dispose(); even if session.end fails.